prow/pod-utils/clone/clone.go

```diff
@@ -109,6 +110,9 @@ func PathForRefs(baseDir string, refs prowapi.Refs) string {
 	var clonePath string
 	if refs.PathAlias != "" {
 		clonePath = refs.PathAlias
+	} else if refs.RepoLink != "" {
+		re := regexp.MustCompile(`^https?://`)
+		clonePath = re.ReplaceAllString(refs.RepoLink, "")
```

Comment:
Do I understand correctly that the purpose of the regex is to strip the `http://` prefix? If so, `strings.TrimPrefix` would be more appropriate
/ok-to-test

Reply:
Yes, that is the intent; however, I used regex instead of `strings.TrimPrefix` in order to remove either `http://` or `https://` since more than likely there are still enterprise deployments without TLS enabled.

Reply:
Sure, so use two `string.Replace`s or just split by `://` and drop the first part? This problem seems a bit too trivial to use a regex